src/auction.js

```diff
@@ -974,3 +977,19 @@ function groupByPlacement(bidsByPlacement, bid) {
   bidsByPlacement[bid.adUnitCode].bids.push(bid);
   return bidsByPlacement;
 }
+
+/**
+ * isValidPrice is price validation function 
+ * which checks if price from bid response
+ * is not higher than top limit set in config
```

Comment:
Can you convert this comment to jsdoc syntax so an ide can pick up for syntax highlighting

Reply:
done